Modules/Devices/src/Devices.Infrastructure/Persistence/Repository/TiersRepository.cs

```diff
@@ -38,6 +44,11 @@ public async Task<DbPaginationResult<Tier>> FindAll(PaginationFilter paginationF
         return paginationResult;
     }
 
+    public async Task<Tier> FindById(TierId tierId, CancellationToken cancellationToken)
+    {
+        return await _tiersDbSet.Include("Identities").SingleAsync(t => t.Id == tierId, cancellationToken);
```

Comment:
Include all, and not just the Identities. 

Plus: if you ever want to include only a single property, do it with a lambda and not with a string. Or at least use the `nameof` operator to get the property name. Otherwise this fails when renaming the property.....